extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/calendar/GoogleCalendarExporter.java

```diff
@@ -40,16 +40,16 @@
 public class GoogleCalendarExporter implements Exporter<AuthData, CalendarContainerResource> {
 
   private volatile Calendar calendarInterface;
-  private TransferStore transferStore; // TODO(olsona): use transferStore
+  private JobStore jobStore;  // TODO(olsona): use jobStore
```

Comment:
Does this comment still apply?

Reply:
Really, we probably don't need a jobstore at this point.